testdata/e2e/e2e_helper.go

```diff
@@ -16,7 +16,7 @@ import (
 	"github.com/prometheus/prom2json"
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	statsapi "k8s.io/kubernetes/pkg/kubelet/apis/stats/v1alpha1"
+	statsapi "k8s.io/kubelet/pkg/apis/stats/v1alpha1"
```

Comment:
These are equivalent packages?  Did it just get moved?

Reply:
Yup, they're equivalent for our purposes. We just use the `Summary` type which is the same in both. From what I was reading, they wanted to move sub packages out of the main kubernetes package - I can't remember what their reasoning was off the top of my head.